src/Schema/Directives/Traits/HasBuilderArgument.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Directives\Traits;
+
+use Illuminate\Contracts\Database\Query\Builder;
+use Laravel\Scout\Builder as ScoutBuilder;
+use Nuwave\Lighthouse\Execution\ResolveInfo;
+use Nuwave\Lighthouse\Support\Contracts\GraphQLContext;
+
+trait HasBuilderArgument
+{
+    /**
+     * @param array<string, mixed> $args
+     * @throws \Nuwave\Lighthouse\Exceptions\DefinitionException
+     */
+    private function getBuilder(mixed $root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo): Builder|ScoutBuilder
```

Comment:
I think `get*` is only appropriate for actual getters (retrieval of an instance property).

```suggestion
    private function builder(mixed $root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo): Builder|ScoutBuilder
```

Reply:
I like a verb for methods, still... So how about resolveBuilder or makeBuilder or getBuilderFromArgs, ... ?

Reply:
`make` seems appropriate.